federatedscope/core/auxiliaries/dataloader_builder.py

```diff
@@ -6,36 +8,52 @@
     Dataset = object
 
 
-def get_dataloader(dataset, config):
+def get_dataloader(dataset, config, split='train'):
     if config.backend == 'torch':
```

Comment:
It seems that there is no reason to indent these lines since we could just return None for the non-torch cases at the beginning.